pkg/client/client_test.go

```diff
@@ -193,15 +201,16 @@ func TestShowTables(t *testing.T) {
 		Limit:  100,
 	}
 
-	c, _ := New(opts)
+	c, err := New(opts)
+	assert.NoError(t, err)
 
 	tables, err := c.ShowTables()
```

Comment:
here you should (must) use

```suggestion
	c, err := New(opts)
	require.NoError(t, err)

	tables, err := c.ShowTables()
```

require is another package available in testify

The reason is that if an error is returned by New, c will be nil, so c.ShowTables will panic

Test should never panic

Reply:
Ok, let's focus on the scope of the PR. We can address the test suite in another PR and issue. This test suite is not new, it's been 3 years since I set it up. I'm willing to improve it but latter.